app/views/stats/repartition.phtml

```diff
@@ -31,74 +34,91 @@
 			<?= _i('configure') ?> <?= _t('gen.action.manage') ?>
 		</a>
 	<?php }?>
+</nav>
+
+<main class="post">
+	<h1><?= _t('admin.stats.repartition', $feedname) ?></h1>
 
-	<div class="stat-grid">
-		<div class="stat table-wrapper scrollbar-thin">
+	<div class="box double-width">
+		<div class="box-title"><h2><?= _t('admin.stats.overview') ?></h2></div>
+		<div class="box-content scrollbar-thin">
 			<table>
 			<tr>
 				<th><?= _t('admin.stats.status_total') ?></th>
-				<th><?= _t('admin.stats.status_read') ?></th>
-				<th><?= _t('admin.stats.status_unread') ?></th>
-				<th><?= _t('admin.stats.status_favorites') ?></th>
+				<th><?= _i('read') ?> <?= _t('admin.stats.status_read') ?></th>
+				<th><?= _i('unread') ?> <?= _t('admin.stats.status_unread') ?></th>
+				<th><?= _i('starred') ?> <?= _t('admin.stats.status_favorites') ?></th>
 			</tr>
 			<tr>
-				<td class="numeric"><?= $this->repartition['total'] ?? -1 ?></td>
-				<td class="numeric"><?= $this->repartition['count_reads'] ?? -1 ?></td>
-				<td class="numeric"><?= $this->repartition['count_unreads'] ?? -1 ?></td>
-				<td class="numeric"><?= $this->repartition['count_favorites'] ?? -1 ?></td>
+				<?php
+				$feedID = $this->feed->id();
```

Comment:
I have no idea how to solve it:
```
Check failure on line 54 in app/views/stats/repartition.phtml
GitHub Actions / tests

Cannot call method id() on FreshRSS_Feed|null.
```

`$this->feed->id();` is used some lines before (line 33) too without any troubles

Reply:
There it's surrounded by `<?php if ($this->feed !== null) {?>`

Reply:
great. Thanks for our help. It solved the issue